packages/contracts/test/PoolMetadataRegistry.t.sol

```diff
@@ -14,22 +17,25 @@ contract MockPoolMetadataRegistry is PoolMetadataRegistry {
     function isPoolRegistered(bytes32 poolId) public view returns (bool) {
         return _isPoolRegistered(poolId);
     }
-
-    function isPoolOwner(bytes32 poolId) public view returns (bool) {
-        return _isPoolOwner(poolId);
-    }
 }
 
 contract PoolMetadataRegistryTest is IPoolMetadataRegistry, Test {
     MockPoolMetadataRegistry _poolMetadataRegistry;
+    MockBasicAuthorizer private _authorizer;
+    MockAuthenticatedContract private _authenticated;
     IVault private _vault;
     MockBasePool private _basePool;
+    MockBasePool private _delegatedBasePool;
     address private constant _poolOwner = address(15);
 
     string private constant _testMetadataCID = "QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR";
+    address internal constant _DELEGATE_OWNER = 0xBA1BA1ba1BA1bA1bA1Ba1BA1ba1BA1bA1ba1ba1B;
+    address DAOmultisig = 0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f;
```

Comment:
I don't think this needs to be a real address - in this case, you're using the DAO multi-sig on Ethereum, but I'd rather generate a random one to avoid confusion.